lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +111,18 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if (not isinstance(unknown_profile, dict) or not isinstance(profile_1, dict) or
+            not isinstance(profile_2, dict)):
+        return None
+    mse_profile_1 = compare_profiles(unknown_profile, profile_1)
+    mse_profile_2 = compare_profiles(unknown_profile, profile_2)
+    if (isinstance(mse_profile_1, float)
```

Comment:
if not - return immediately

Reply:
I didn't understand. Could you explain please?

Reply:
Fixed